var/spack/repos/builtin/packages/wannier90/package.py

```diff
@@ -76,10 +174,17 @@ def install(self, spec, prefix):
             join_path(self.prefix.bin, 'postw90.x')
         )
 
-        install(
-            join_path(self.stage.source_path, 'libwannier.a'),
-            join_path(self.prefix.lib, 'libwannier.a')
-        )
+        if '+shared' in spec:
+            if spec.satisfies('@3:'):
+                inst = ['libwannier.a', 'libwannier.so']
+            else:
+                inst = ['libwannier.so']
```

Comment:
Is a `.so` file also installed on macOS, or is it a `.dylib` file?

Reply:
@adamjstewart: good catch!  I'm guessing you're right on MacOS!
I'm not sure if have a good MacOS box to test on!
"if self.spec.satisfies('platform=darwin'): ..."
Would this work to test if I'm on MacOS?

Reply:
Yes, that's the right test for whether you're on macos.